test/test_minimal.cpp

```diff
@@ -0,0 +1,6 @@
+#define JC_TEST_USE_DEFAULT_MAIN
+#include <jc_test.h>
+
+TEST_CASE(multiply) {
+  CHECK_EQ(4, 2*2);
+}
```

Comment:
Not a huge deal, but currently this file tests two things:
* `JC_TEST_USE_DEFAULT_MAIN`
* Doctest api

Would you mind adding a `test_doctest.cpp` for the doctest api?

Reply:
Let me rename ```test_minimal.cpp``` to ```test_doctest.cpp```.